java-extensions/pom.xml

```diff
@@ -207,65 +209,65 @@
                     </execution>
                 </executions>
             </plugin>
-            <!-- copy config files from fe before checkstyle -->
```

Comment:
Why remove this plugin?

Reply:
Zhengjin: This plugin is for code style check, it is too strict for the code added for Pixels integration.